microfragments/src/main/java/org/dmfs/android/microfragments/transitions/BackTransition.java

```diff
@@ -110,7 +110,15 @@ public Timestamp timestamp()
     @Override
     public void prepare(@NonNull Context context, @NonNull FragmentManager fragmentManager, @NonNull MicroFragmentHost host, @NonNull MicroFragment<?> previousStep)
     {
-        mResponseCage.pigeon(fragmentManager.popBackStackImmediate()).send(context);
+        boolean result = fragmentManager.popBackStackImmediate();
```

Comment:
Shouldn't this come after popping the skip steps? So if only "skip"s are on the backstack, then it should be `false`, right?